### PR TITLE
feature: use the fetchJsonLd in json-ld.js as documentLoader to pass authentification headers

### PR DESCRIPTION
**Pull request by Simperfit**

### CODE REVIEW

File: src/hydra/fetchJsonLd.js [hunk 1]
```diff
@@ -16,6 +16,11 @@ export default function fetchJsonLd(url, options = {}) {
     options.headers.set('Accept', jsonLdMimeType);
   }
 
+  if (null === options.headers.get('Authorization')) {
```
Comment (dunglas) [inline src/hydra/fetchJsonLd.js]: I don't think it's the good way to handle this. You should use a decorator as explained here IMO: https://github.com/api-platform/admin#add-authentication-support

Comment (Simperfit) [inline src/hydra/fetchJsonLd.js]: It does not satisfy the needs just using the decorator. (or I don't have the good version atm).

**Comment by Simperfit** [comment]
Thanks to @harijoe for the help on this ;)

File: src/hydra/fetchJsonLd.js [hunk 2]
```diff
@@ -26,6 +26,6 @@ export default function fetchJsonLd(url, options = {}) {
         return {response: response};
       }
 
-      return response.json().then(body => { return { response, body }});
+      return response.json().then(body => { return { response, body, document: body }});
```
Comment (dunglas) [inline src/hydra/fetchJsonLd.js]: Why this change?

**Comment by dunglas** [comment]
Thanks @Simperfit
